django_project/core/settings/project.py

```diff
@@ -49,6 +49,7 @@
         'js/localitySidebar.js',
         'js/map.js',
         'js/app.js',
+        'js/bootstrap.min.js',
```

Comment:
so there are two problems with this line and Irwan:

1) this is project.py so we should only have things in here that we ourselves have written
2) bootstrap.css should be coming from CDN so we should not be putting it in pipeline